dask/array/core.py

```diff
@@ -19,6 +19,7 @@
 from . import chunk
 from .slicing import slice_array
 from . import numpy_compat
+from ..base import DaskBase
```

Comment:
Perhaps just `Base` and leave the name `dask` to the library?  Outsiders could refer to this just as `dask.Base`.

Until `dask.imperative` came around this had the name `Collection` in my head.